lib/fizzy/cxx20/bit.hpp

```diff
@@ -0,0 +1,26 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <type_traits>
+
+namespace fizzy
```

Comment:
Oh actually could have added the C++20 check as in `span.hpp`, to test this with a C++20 compiler.